pom.xml

```diff
@@ -154,67 +154,62 @@
       <groupId>com.vaadin</groupId>
     </dependency>
 
+    <!-- Added to provide logging output as Flow uses -->
+    <!-- the unbound SLF4J no-operation (NOP) logger implementation -->
+    <dependency>
+      <artifactId>slf4j-simple</artifactId>
+      <groupId>org.slf4j</groupId>
+    </dependency>
+
+    <!-- Java EE-->
     <dependency>
       <artifactId>javaee-api</artifactId>
       <groupId>javax</groupId>
       <scope>provided</scope>
-      <version>7.0</version>
+      <version>${javaee-api.version}</version>
     </dependency>
 
-    <!-- Added to provide logging output as Flow uses -->
-    <!-- the unbound SLF4J no-operation (NOP) logger implementation -->
+    <!-- Database -->
     <dependency>
-      <artifactId>slf4j-simple</artifactId>
-      <groupId>org.slf4j</groupId>
+      <artifactId>postgresql</artifactId>
+      <groupId>org.postgresql</groupId>
+      <version>${postgresql.version}</version>
     </dependency>
 
     <!-- Lombok -->
     <dependency>
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
-      <version>1.18.10</version>
+      <version>${lombok.version}</version>
       <scope>provided</scope>
     </dependency>
 
-    <!-- JUnit5 -->
+    <!-- Other-->
     <dependency>
-      <artifactId>junit-jupiter-engine</artifactId>
-      <groupId>org.junit.jupiter</groupId>
-      <scope>test</scope>
-      <version>${junit-jupiter-engine.version}</version>
+      <artifactId>vt-password</artifactId>
+      <groupId>edu.vt.middleware</groupId>
+      <version>${vt-password.version}</version>
     </dependency>
 
     <dependency>
-      <artifactId>junit-platform-runner</artifactId>
-      <groupId>org.junit.platform</groupId>
-      <version>${junit-platform-runner.version}</version>
-      <scope>test</scope>
+      <artifactId>owner</artifactId>
+      <groupId>org.aeonbits.owner</groupId>
```

Comment:
Why we need that library? What alternatives do we have? What other alternatives are available?